lib/worker_tools/basics.rb

```diff
@@ -93,7 +89,6 @@ def find_model
       @model_id ||= nil
       return @model_id if @model_id.is_a?(model_class)
       return model_class.find(@model_id) if @model_id
-      raise 'Model not available' unless create_model_if_not_available
```

Comment:
why are you throwing this out?

Reply:
We had no real cases where the model should not be created as a fallback.  The default `false` lead to us to define this method with `true` all over the place. I think this was related to some case where the modules were meant to work just on memory without a persistence layer, but it does not longer apply.  In that case you would still need a AR compatible object / class with repository pattern, and it would be easier to define `model` altogether.